ui/src/components/MotorInput/MotorInput.jsx

```diff
@@ -55,32 +69,34 @@ function MotorInput(props) {
             state={state}
             precision={precision}
             step={step}
-            testId={`MotorInput_value_${motorName}`}
+            testId={`MotorInput_value_${attribute}`}
             disabled={disabled}
-            onChange={(val) => save(motorName, val)}
+            onChange={(val) => dispatch(setAttribute(attribute, val))}
           />
         </div>
-        {saveStep &&
```

Comment:
Worth pointing out that I'm able to remove this condition because `OneAxisTranslationControl` no longer uses `MotorInput` since #1445 (it uses the lower-level component `BasicMotorInput`).